server.go

```diff
@@ -36,6 +36,11 @@ import (
 	"github.com/globalsign/mgo/bson"
 )
 
+const (
+	defaultWriteBatchSize      = 1000
```

Comment:
This is 100000 in version 3.6.
https://docs.mongodb.com/master/reference/limits/#Write-Command-Batch-Limit-Size
This should be safe to change, as defaultWriteBatchSize is used only for the OP_MSG right now.